profiles/arts_signal_scalp_trading/profile.json

```diff
@@ -0,0 +1,43 @@
+{
+    "config": {
+        "crypto-currencies": {
+            "Bitcoin": {
+                "enabled": true,
+                "pairs": [
+                    "BTC/USDT"
+                ]
+            }
+        },
+        "exchanges": {
+            "binance": {
+                "enabled": true
+            }
+        },
+        "trader": {
+            "enabled": false,
+            "load-trade-history": false
+        },
+        "trader-simulator": {
+            "enabled": true,
+            "fees": {
+                "maker": 0.1,
+                "taker": 0.1
+            },
+            "starting-portfolio": {
+                "BTC": 10,
+                "USDT": 1000
+            }
+        },
+        "trading": {
+            "reference-market": "USDT",
+            "risk": 1
+        }
+    },
+    "profile": {
+        "avatar": "default_profile.png",
+        "description": "Arts Signal Scalp trading connects OctoBot to Art's GodBot scalp telegram signals.",
```

Comment:
Should we include the telegram group url ?

Reply:
Yes :+1: